### DISCUSSION TITLE
HA Interfacing

### DISCUSSION THREAD

**Discussion by whiteas2** [spbrogan/sensor.mopeka_pro_check#12] [General]
Hi. 

I am relatively new to HA so excuse me if silly question.

I have a rebranded mopeka pro sensor.
I have it working with mobile app.
I have followed installation instructions on HA.
I have tried the 'call service' function to get the MAC address and checked under states but there does not seem to be any info.

1) I was not able to determine from documentation if this sensor will work directly to RPI4 via BT? or if I need to order the wifi bridge as well?
2) If I dont need the wifi bridge; do I need to do something on the RPI for this to work? eg activate BT?

thank you!

**Comment by spbrogan** [comment]
assuming the rpi4 has working bluetooth, your sensor is within 10 feet (probably more like 100 feet but let's go within 10 to be safe) and you are following the directions here it should work: https://github.com/spbrogan/sensor.mopeka_pro_check#setting-up-a-new-sensor

Did you press the green button 10 times to wake it up?  I don't remember if you had to also hold in the green button while it is scanning (you should try that too).  

If that still doesn't work, we would need to look closer at the BLE advertisements.  It seems that every "new" version of mopeka sensor they may change the ble advertisement.  

You do not need the wifi bridge as that is in no way involved.

**Comment by whiteas2** [comment]
Thank you for the quick response! I had the sensor within 10 feet. I did follow directions and pressed the button 10 times. Let me play around a bit... at least now I know it does not need the bridge. I will let you know if I come right or not.

**Comment by whiteas2** [comment]
Sorted :-). I had to press the button 10 times quickly; and then hold. Working great. Thank you for the assistance
